model/vm_host.rb

```diff
@@ -22,6 +22,8 @@ class VmHost < Sequel::Model
   include HealthMonitorMethods
   semaphore :checkup, :reboot, :destroy
 
+  SSH_HEALTH_CHECK_COMMAND = "RFILE=/tmp/ubiiotest-$RANDOM;timeout 10 dd if=/dev/urandom of=$RFILE bs=1M count=10;rm $RFILE"
```

Comment:
I recommend we check that we can read back what we wrote.  We can write a 10MB file from `/dev/zero`, and then check the SHA256 of the file is `e5b844cc57f57094ea4585e235f36c78c1cd222262bb89d53c94dcb4d6b3e55d`.

Reply:
I had given a thought to write something known (like a hard-coded string) to be able to read back, but that could be too short. Haven't considered the zeroes approach before. I have to admit, it sounds slightly awkward to write just zeroes (no real reason, just my hunch), but since you said it, let's go with it.